src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+@use "./variables" as val;
+@use "sass:map";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  position: fixed;
+  width: 780px;
```

Comment:
The calendar width and the actual width don't match. Consider fixing it 
<img width="940" alt="Screenshot 2024-06-23 at 15 44 49" src="https://github.com/mate-academy/layout_calendar/assets/135232001/5b2e5568-318b-49e9-be19-923f196dc0fc">

Reply:
wow thanks!